packages/client/src/net/peerpool.ts

```diff
@@ -191,7 +191,7 @@ export class PeerPool {
     // Reconnect to peer after ban period if pool is empty
     this._reconnectTimeout = setTimeout(async () => {
       if (this.running && this.size === 0) {
-        await peer.server?.connect(peer.id)
+        await peer.connect()
```

Comment:
What does this do?  Looks like this is a noop added in `peer.ts`?

Reply:
It was a noop before (see connect() method in server, now it's calling into the existing connect() from RlpxPeer.